catlin/pkg/validator/helper.go

```diff
@@ -0,0 +1,93 @@
+// Copyright © 2021 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validator
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/spf13/viper"
+)
+
+// Checks if the added category annotation is
+// from predefined category list
+func contains(slice []string, val string) bool {
+	for _, item := range slice {
+		if item == val {
+			return true
+		}
+	}
+	return false
+}
+
+// Loads the Env variable
+func LoadEnv() string {
+	viper.SetConfigFile(".env")
+
+	// Find and read the config file
+	err := viper.ReadInConfig()
+	if err != nil {
+		viper.SetDefault("CONFIG_FILE_URL", "https://raw.githubusercontent.com/tektoncd/hub/main/config.yaml")
```

Comment:
you can just return the file url from here.
instead of setdefault and getting again